lib/assert.js

```diff
@@ -59,31 +59,29 @@ function wrapAssertions(callbacks) {
 		},
 
 		is(actual, expected, message) {
-			if (actual === expected) {
+			if (Object.is(actual, expected)) {
 				pass(this);
 			} else {
 				const diff = formatAssertError.formatDiff(actual, expected);
 				const values = diff ? [diff] : [
 					formatAssertError.formatWithLabel('Actual:', actual),
-					formatAssertError.formatWithLabel('Must be strictly equal to:', expected)
+					formatAssertError.formatWithLabel('Must be strictly equal (using Object.is) to:', expected)
```

Comment:
Perhaps we can take a leaf from the [`Object.is()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is) description and say `Must be the same value as:`.